app/ante/vesting.go

```diff
@@ -79,14 +78,6 @@ func (vtd EthVestingTransactionDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx,
 				"cannot perform Ethereum tx with clawback vesting account, that has no vested coins: %s", vested,
 			)
 		}
-
-		// Error if account has locked coins (before surpassing all lockup periods)
-		islocked := clawbackAccount.HasLockedCoins(ctx.BlockTime())
```

Comment:
`locked` doesn't mean the same on Evmos and the SDK. So if the tests are correct, I think they should fail when you remove this. 

**Evmos**
`HasLockedCoins` uses `GetUnlockedOnly` and determines the amount of `locked` coins by reading the locking schedule of a `ClawbackVestingAccount`.

**Cosmos SDK**
In the x/bank module `SendCoins` uses `subUnlockedCoins` ([here](https://github.com/cosmos/cosmos-sdk/blob/main/x/bank/keeper/send.go#L192)) to determine if a vesting account has sufficient funds. It checks the amount of `LockedCoins`of a `VestingAccount`, however here `locked` is defined as 
> the vesting coins that are not delegated.

Reply:
Thanks for the clarification @danburck. The Evmos `ClawbackAccount` calculates `LockedCoins` to include Evmos' locking schedule by way of `GetVestedCoins`:

```
// GetVestedCoins returns the total number of vested coins that are still in lockup. If no coins are
// vested, nil is returned.
func (va ClawbackVestingAccount) GetVestedCoins(blockTime time.Time) sdk.Coins {
	// It's likely that one or the other schedule will be nearly trivial,
	// so there should be little overhead in recomputing the conjunction each time.
	coins := va.GetUnlockedOnly(blockTime).Min(va.GetVestedOnly(blockTime))
	if coins.IsZero() {
		return nil
	}
	return coins
}
```.

As a result, we can safely rely on the `bank` module's accounting using `LockedCoins`, which this implementation does.

Reply:
@austinchandra makes sense, can you make this more obvious in the comment that `sendCoins` in the x/bank keeper checks for LockedCoins to determine spendable coins?  On Evmos, coins are not spendable if they are locked  OR vesting coins that are not delegated.